frontend-vite/src/App.jsx

```diff
@@ -6,15 +6,19 @@ import Home from "./pages/Home";
 import Login from "./pages/Login";
 import Profile from "./pages/Profile";
 import CreateProfile from "./pages/CreateProfile";
+import NavBar from "./components/NavBar";
+import UpdateProfile from "./pages/UpdateProfile";
 
 function App() {
   return (
     <Routes>
       <Route exact path="/" element={<Login />} />
-      <Route exact path="/register" element={<CreateProfile />} />
+      <Route exact path="//register" element={<CreateProfile />} />
+      <Route exact path="/navbar" element={<NavBar />} />
```

Comment:
do we need a route for just rendering navbar?

Reply:
No, I don't think so. Should I delete it?